styleguide/derek/assets/forms.ejs

```diff
@@ -42,6 +42,32 @@
       </div>
     </div>
     <div class="form-group row">
+      <div class="col-md-12">  
+        <label>Checkboxes</label>
+        <div class="form-checkboxes form-control">
+          <div class="form-item">
+            <input type="checkbox" name="checkbox-0" value="0" class="form-checkbox">  <label class="option" for="checkbox-0">Checkbox 0</label>
+          </div>
+          <div class="form-item">
+            <input type="checkbox" name="checkbox-1" value="1" class="form-checkbox">  <label class="option" for="checkbox-1">Checkbox 1</label>
```

Comment:
Label `for` attributes should reference a corresponding `id` attribute in the `input` tag, so when you click on the label, it auto focuses on the input field. This comment applies to all instances of label on this page.